### PR TITLE
Provide container image to improve ease-of-use for interested parties

### PR DESCRIPTION
**Pull request by lmeinen**
This pull-request adds a new controller, which provides a simple API similar to the iOS/Androi SDKs. It also provides a Docker container image.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
+FROM adoptopenjdk/openjdk11
+
+LABEL maintainer="Lasse Meinen <meinen@ubique.ch>"
+
+# Note: Newest release doesn't include simple controller yet
+ARG VERSION=v2.6.0-prerelease
```
Comment (martinalig) [inline Dockerfile:6]: Version nicht hardcoden. Man kann Build Args übergeben:
`--build-arg VERSION=v2.6.0-prerelease`

File: README.md [hunk 2]
```diff
@@ -12,6 +12,54 @@ The app design, UX and implementation was done by [Ubique](https://www.ubique.ch
 This service provides an API, which is consumed by other CovidCertificate backend services. It offers an endpoint to allow the decoding and verification of a given Covid Certificate.
 It regularly sends a request to the [Verifier Service](https://github.com/admin-ch/CovidCertificate-App-Verifier-Service) to keep an up-to-date list of trusted DSCs with which Covid Certificates could be signed.
 
+## Simple Verification
+
+In order to allow the independent usage of the Verification-Check-Service without deeper knowledge of the source code, the repository includes a Dockerfile to start the service in a separate container. However, one needs to be in possession of the Verifier-Service's baseurl and a corresponding api-token to use the service.
```
Comment (martinalig) [inline README.md:17]: Base URLs sind bekannt, bitte hier auflisten:
* ABN: https://www.cc-a.bit.admin.ch
* DEV: https://www.cc-d.bit.admin.ch
* PROD: https://www.cc.bit.admin.ch

File: README.md [hunk 3]
```diff
@@ -12,6 +12,54 @@ The app design, UX and implementation was done by [Ubique](https://www.ubique.ch
...
+
+### Starting the service
+
+To build the image, clone the repository and run
+```bash
+docker build -t simple-verification CovidCertificate-App-Verification-Check-Service
```
Comment (martinalig) [inline README.md:23]: Hat mich ein wenig verwirrt. Besser in den Folder cden und dann mit `.` ausführen

File: ch-covidcertificate-backend-verification-check/ch-covidcertificate-backend-verification-check-ws/src/main/java/ch/admin/bag/covidcertificate/backend/verification/check/ws/controller/SimpleController.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@Profile("simple")
+@RestController
+@RequestMapping("simple")
```
Comment (martinalig) [inline ch-covidcertificate-backend-verification-check/ch-covidcertificate-backend-verification-check-ws/src/main/java/ch/admin/bag/covidcertificate/backend/verification/check/ws/controller/SimpleController.java:28]: Versionieren: `/v1/simple`

**Review by ubhaller — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -12,6 +12,54 @@ The app design, UX and implementation was done by [Ubique](https://www.ubique.ch
...
+```
+
+Then start the container by running
+```bash
+docker run --name simple-verification -e BASEURL=... -e APIKEY=... simple-verification
```
[the remark this answers is not in the document]
Comment (martinalig) [reply README.md:28]: @UBaggeler Should be able to help here :)

File: README.md (hunk 2 above)
Comment (lmeinen) [reply README.md]: See Felix's comment regarding setting the baseurl based on the profile

File: ch-covidcertificate-backend-verification-check/ch-covidcertificate-backend-verification-check-ws/simple-config/simple-verification-ws.properties [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+# SPDX-License-Identifier: MPL-2.0
+#
+
+# Change profile "detailed" to "simple" to use the SimpleController (i.e. simplified verify response)
+spring.profiles.active=prod,simple
+management.endpoints.enabled-by-default=false
+server.error.whitelabel.enabled=true
+
+# Baseurl and api-key are passed as env variables upon starting container
+verifier.baseurl=${BASEURL}
```
[the remark this answers is not in the document]
Comment (lmeinen) [reply ch-covidcertificate-backend-verification-check/ch-covidcertificate-backend-verification-check-ws/simple-config/simple-verification-ws.properties]: I solved it now with separate property files. Is there a prettier way to do this?

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (UBaggeler) [reply README.md]: Let's try to keep it in sync with what we write for the Android/[iOS SDKs](https://github.com/admin-ch/CovidCertificate-SDK-iOS/blob/main/README.md):

> The SDK needs to be initialized with an environment and a API token. 
This allows for different verification rules per environment or other environment-specific settings.
> If you intend to integrate the CovidCertificate-SDK-iOS into your app, please get in touch with the [BAG](mailto:Covid-Zertifikat@bag.admin.ch) to get a token assigned.

File: README.md [hunk 7]
```diff
@@ -12,6 +12,55 @@ The app design, UX and implementation was done by [Ubique](https://www.ubique.ch
...
 
+## Simple Verification
+
+In order to allow the independent usage of the Verification-Check-Service without deeper knowledge of the source code, the repository includes a Dockerfile to start the service in a separate container. However, one needs to be in possession of an api-token to access the Verifier-Service.
+
+### Starting the service
+
+To build the image, clone the repository, `cd` into the cloned directory and run
+```bash
+docker build -t simple-verification .
```
Comment (UBaggeler) [inline README.md:23]: I would use `CovidCertificate-App-Verification-Check-Service` as name, instead of `simple-verification`

**Review by ubamrein — APPROVED** [review]

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (ubamrein) [reply README.md]: @UBaggeler @gstoehld is this still relevant?

**Review by UBaggeler — APPROVED** [review]

**Review by UBaggeler — APPROVED** [review]

**Review by ubhaller — APPROVED** [review]

**Pull request merged**